README.md

```diff
@@ -279,6 +279,8 @@ Detailed contribution instructions can be found in the [taxonomy repository](htt
 
    The synthetic data set will be three files in the newly created `generated` directory named `generated*.json`, `test*.jsonl`, and `train*.jsonl`.
 
+   > **NOTE:** If you want to pickup from where a failed or canceled `ilab generate` left off, you can copy the `generated*.json` file into a file named `regen.json`. `regen.json` will be picked up at the start of `lab generate` when available. You should remove it when the process is completed.
+
```

Comment:
```suggestion
   > [!NOTE] 
   > If you want to pickup from where a failed or canceled `ilab generate` left off, you can copy the 
   > `generated*.json` file into a file named `regen.json`. `regen.json` will be picked up at the start of `lab 
   > generate` when available. You should remove it when the process is completed.
```

Reply:
that does look better but now its not the same style as other notes in this file

Reply:
Well maybe another pr can convert the rest 😀